components/blitz/src/omero/gateway/facility/AdminFacility.java

```diff
@@ -60,6 +61,9 @@
     /** Reference to the roles.*/
     private Roles roles;
 
+    /** All available admin privileges */
+    private Collection<String> adminPrivileges;
```

Comment:
If we're reusing this (which we should), maybe safest to make it immutable.

Reply:
@dominikl do you want to fix it?

Reply:
Already done in commit below ace75ec .

Reply:
thanks